build/shade/EnvironmentParameters.shade

```diff
@@ -0,0 +1,20 @@
+use namespace='System'
+ 
+functions @{
+    static string GetEnvironmentParameter(string name)
+    {
+        return GetEnvironmentParameter(name, s => s);
+    } 
+
+    static T GetEnvironmentParameter<T>(string name, Func<string, T> valueFn)
+    {
+        var value = valueFn(Environment.GetEnvironmentVariable(name));
+        Console.WriteLine(name + ": " + (value != null ? value.ToString() : "(null)"));
```

Comment:
I hadn't given it proper thought, but it actually works. I'm calling this with a value type (`bool`) to set `skipNpCredentials`.